### DISCUSSION TITLE
Radar prmt build in local system

### DISCUSSION THREAD

**Discussion by this-Aditya** [RADAR-CNS/radar-prmt-android#124] [Q&A]
Hello, I just forgot about the to copy what joris has told me to add yesterday from chatbox.                                                            
Currently I am getting these errors 
1. Could not resolve all files for configuration ':app:playStoreDebugRuntimeClasspath'.(I don't know which dependency is missing )
2.No matching client found for package name 'org.radarcns.detail'
3. Could not find org.radarbase:radar-commons-android:1.2.2. (Even After Adding     api 'org.radarbase:radar-commons-android:1.2.2')

**Comment by dmdbilal** [comment]
For 2nd error, change the package_name in debug\google-service.json from `com.radarcns.android` to `com.radarcns.detail`. This resolved my error.

**Comment by dmdbilal** [comment]
I'm getting this error, `Could not find com.google.firebase:firebase-core:.` have you got this error during build.
